### PR TITLE
TST: Install sphinx-astropy from pip

### PR DESCRIPTION
**Pull request by pllim**
Close #512

### CODE REVIEW

File: .travis.yml
Comment (pllim) [inline .travis.yml:32]: Not sure what this is meant to test. Seems a little redundant, but I can add it back if you want.

**Comment by pllim** [comment]
What do you think, @mwcraig ?

**Comment by embray** [comment]
This makes sense to me.  Is there anything else holding this up?  I can give it a positive review.  (By the way, it seems I do not have the right access levels on this repo; I cannot assign reviewers/labels/etc.)

**Review by mwcraig — APPROVED** [review]
This looks good to me; sorry for the long delay in reviewing it.

**Comment by pllim** [comment]
I cannot merge it either. Maybe need to ping @astropy/coordinators to give Madison the proper access.

**Comment by astrofrog** [comment]
See https://github.com/astropy/astropy-helpers/issues/512 for a quick question before merging

**Review by astrofrog — COMMENTED** [review]
See https://github.com/astropy/astropy-helpers/pull/515 for an easier alternative (if it works)

**Comment by astrofrog** [comment]
@embray - I've added you to this repository **but** note that I think #515 might make more sense.

**Pull request closed without merging**
